Resources/Prototypes/_Sunrise/Entities/Effects/overlays.yml

```
@@ -0,0 +1,10 @@
+- type: entity
+  id: EffectNightVision
+  categories: [ HideSpawnMenu ]
+  name: night vision
+  components:
+  - type: PointLight
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)